backend/Dockerfile.prod

```
@@ -0,0 +1,67 @@
+ENV DEBUG=False
+
+# Install runtime dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    default-libmysqlclient-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Create a non-root user to run the app
+RUN useradd -m appuser
+RUN chown -R appuser:appuser /app
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)